### PR TITLE
Add CoinGecko IDs to several assetlist.json files

### PR DESCRIPTION
**Pull request by blake-regalia**
This PR updates several `assetlist.json` files with the correct corresponding `coingecko_id` for several assets, manually verified by hand.

### CODE REVIEW

File: injective/assetlist.json [hunk 1]
```diff
@@ -143,6 +144,7 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/ninja.png"
       },
+      "coingecko_id": "dog-wif-nuchucks",
```
Comment (JeremyParish69) [inline injective/assetlist.json:147]: I wonder if they intend on correcting this incorrect spelling of nunchucks (missing the second 'n')...

File: osmosis/assetlist.json [hunk 2]
```diff
@@ -1105,6 +1107,7 @@
           }
         }
       ],
+      "coingecko_id": "pstake-staked-osmo",
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:1110]: Please remove this cgid (we only want them defined in the assetlist corresponding to the (usually just one) chain they are originally issued).

File: osmosis/assetlist.json [hunk 3]
```diff
@@ -1302,6 +1305,7 @@
           }
         }
       ],
+      "coingecko_id": "autism",
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:1308]: Please remove this cgid (we only want them defined in the assetlist corresponding to the (usually just one) chain they are originally issued).

File: osmosis/assetlist.json (hunk 2 above)
Comment (blake-regalia) [reply osmosis/assetlist.json]: fixed

File: osmosis/assetlist.json (hunk 3 above)
Comment (blake-regalia) [reply osmosis/assetlist.json]: fixed

File: injective/assetlist.json (hunk 1 above)
Comment (blake-regalia) [reply injective/assetlist.json:147]: i have no idea. want me to remove it for now?

**Pull request merged**